pom.xml

```diff
@@ -69,10 +81,11 @@
 			<version>${kettle.version}</version>
 			<scope>provided</scope>
 		</dependency>
+		<!-- Test -->
 		<dependency>
 			<groupId>pentaho-kettle</groupId>
 			<artifactId>kettle-engine-test</artifactId>
-			<version>${kettle.version}</version>
+			<version>7.1.0.32-246</version>
```

Comment:
Any reason this is not the same as "kettle.version" ?

Reply:
I don't know why, but this is the last official version released for kettle-engine-test.

Reply:
Then it should be set under `kettle.version` variable.